generator/config/stage/project.yaml

```diff
@@ -10,9 +10,6 @@ arguments:
     -
         name: specification
         type:
-            - bool
-            - int
+            - expression
```

Comment:
How is this change related to the operators in the PR title?

Is it correct to include `expression` here? Looking at the [`$project` syntax](https://www.mongodb.com/docs/manual/reference/operator/aggregation/project/#syntax), an expression is only supported as a _value_ for a field. Or am I misunderstanding and `type` here is only applying to the values for the keys/properties within the specification document?

Reply:
In the [`$lastN` example](https://www.mongodb.com/docs/manual/reference/operator/aggregation/lastN/#using--lastn-as-an-aggregation-expression), the operator is used in the `$project` stage. That's why I had to fix it.
You can see how it's used in [`Expression\LastNOperatorTest::testUsingLastNAsAnAggregationExpression`](https://github.com/mongodb/mongo-php-builder/pull/14/files#diff-d5bb1a8a1c567e077656e5f8948ef77bee8f0c04ca65775bf9586a913f45ed05R37)

Reply:
Then, to confirm, I assume `type` here is just referring to the _values_ of the `specification` object. I misunderstood and assumed this might allow `specification` to be an `expression` directly.

Reply:
Yes, that's the nuance with variadic arguments. The type applies to the values below the root.